create_manifest.py

```diff
@@ -0,0 +1,97 @@
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+/*******************************************************************************
+"""
+
+import sys
+import argparse
+import tomli_w
+
+def auto_int(x):
+	return int(x, 0)
+
+def get_argparser():
+	parser = argparse.ArgumentParser(description="Load an app onto the device from a hex file.")
+	parser.add_argument("-output", help="File where the manifest is saved", required=True)
+	parser.add_argument("--targetId", help="The device's target ID (default is Ledger Blue)", type=auto_int, default=0x31000002)
+	parser.add_argument("--apiLevel", help="Set the API level of the SDK used to build the app", type=auto_int, default=0)
+	parser.add_argument("--fileName", help="The application hex file to be loaded onto the device", required=True)
+	parser.add_argument("--curve", help="""A curve on which BIP 32 derivation is locked ("secp256k1", "secp256r1", "ed25519" or "bls12381g1"), can be repeated""", action='append')
+	parser.add_argument("--path", help="""A BIP 32 path to which derivation is locked (format decimal a'/b'/c), can be repeated""", action='append')
+	parser.add_argument("--appName", help="The name to give the application after loading it", required=True)
+	parser.add_argument("--appFlags", help="The application flags", type=auto_int, default=0)
+	parser.add_argument("--dataSize", help="The code section's size in the provided hex file (to separate data from code, if not provided the whole allocated NVRAM section for the application will remain readonly.", type=auto_int)
+	parser.add_argument("--appVersion", help="The application version (as a string)")
+	parser.add_argument("--gif", help="The gif icon")
+
+	# Not used - to be compatible with python3 ledgerblue
+	parser.add_argument("--icon", help="The icon content to use (hex encoded)")
+	parser.add_argument("--targetVersion", help="Set the chip target version")
+	parser.add_argument("--path_slip21", help="""A SLIP 21 path to which derivation is locked""", action='append')
+	parser.add_argument("--signature", help="A signature of the application (hex encoded)")
+	parser.add_argument("--signApp", help="Sign application with provided signPrivateKey", action='store_true')
+	parser.add_argument("--bootAddr", help="The application's boot address", type=auto_int)
+	parser.add_argument("--rootPrivateKey", help="""The Signer private key used to establish a Secure Channel (otherwise
+a random one will be generated)""")
+	parser.add_argument("--signPrivateKey", help="Set the private key used to sign the loaded app")
+	parser.add_argument("--apdu", help="Display APDU log", action='store_true')
+	parser.add_argument("--deployLegacy", help="Use legacy deployment API", action='store_true')
+	parser.add_argument("--apilevel", help="Use given API level when interacting with the device", type=auto_int, default=10)
```

Comment:
This parameter has been removed from loadApp.py and could therefore be removed here too.

Reply:
Which one ? 
I see all the these element in [ledgerctl/loadApp](https://github.com/LedgerHQ/blue-loader-python/blob/master/ledgerblue/loadApp.py) but not in the same order

Reply:
`apilevel` has been removed, but `apiLevel` has been kept (https://github.com/LedgerHQ/blue-loader-python/pull/100)

Reply:
Thanks done